charts/templates/cluster-global-sidecar.yaml

```diff
@@ -0,0 +1,278 @@
+{{ if .Values.component }}
+{{ if .Values.component.globalSidecar }}
+{{ if .Values.component.globalSidecar.enable }}
+{{ if eq .Values.component.globalSidecar.type "cluster" }}
+{{ $gs := .Values.component.globalSidecar }}
+  {{ range $.Values.module }}
+  {{ if .fence }}
+  {{ $f := .fence }}
+  {{ $g := .global }}
+  {{ $name := .name }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: global-sidecar
+  namespace: {{ $.Values.namespace }}
+  labels:
+    app: global-sidecar
+    service: global-sidecar
+spec:
+  ports:
+    {{- range .fence.wormholePort }}
+    - name: http-{{ . }}
+      port: {{ int . }}
+      protocol: TCP
+      targetPort: 80
+    {{- end }}
+  selector:
+    app: global-sidecar
+  sessionAffinity: None
+  type: ClusterIP
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: global-sidecar
+  namespace: {{ $.Values.namespace }}
+  labels:
+    account: global-sidecar
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: global-sidecar
+  namespace: {{ $.Values.namespace }}
+  labels:
+    app: global-sidecar
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: global-sidecar
+  template:
+    metadata:
+      annotations:
+        proxy.istio.io/config: |
+          proxyMetadata:
+            ISTIO_META_LAZYLOAD_APP:
+              GLOBAL_SIDECAR
+            ISTIO_META_ISTIO_VERSION:
+              "999.0.0"
+        {{- if $g }}
+        {{- if $g.misc }}
+        {{- if $g.misc.metric_source_type }}
+        {{- if (eq $g.misc.metric_source_type "accesslog") }}
+        sidecar.istio.io/bootstrapOverride: "lazyload-accesslog-source"
+        {{- end }}
+        {{- end }}
+        {{- end }}
+        {{- end }}
+        {{- if $f.globalSidecar }}
+        {{- if $f.globalSidecar.excludeInbound }}
+        traffic.sidecar.istio.io/excludeInboundPorts: {{ $f.globalSidecar.excludeInboundPorts }}
```

Comment:
`excludeInbound` or `excludeInbounds`? And I think maybe instead of having the user explicitly specify this port, we can use `containerPort` and the user can just tell us if inbound interception is enabled.

Reply:
as inbounds traffic control is necessary in accesslog metric mode, I remove the config of disable `excludeInbound`